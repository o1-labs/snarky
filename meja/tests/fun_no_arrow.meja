let x = fun (x) {
  x;
};
