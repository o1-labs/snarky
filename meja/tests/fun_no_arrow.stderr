File "tests/fun_no_arrow.meja", line 1, characters 16-17:
Error: Expected => before {

